### PR TITLE
[NEW UI] Fix storybook components and pages

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      |  Fix storybook components and pages because some of the variables has been renamed
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | -

### CODE REVIEW

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by ga-devfront — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
